src/utils/index.js

```diff
@@ -117,7 +117,6 @@ exports.flattenData = (data, xAccessor, yAccessor) => {
         },
         d: item,
         id: series.name + j,
-        series: series,
```

Comment:
This inclusion is intentional and should not be removed. It allows functions iterating over these points to have access to custom props passed to the entire series that are not available at the point level. For example, I might want to color my points based on a custom prop `myColorProp` that I attach to the series object.